examples/compass.py

```diff
@@ -1,7 +1,16 @@
+"""
+    compass.py
+    ~~~~~~~~~~
+
+    Creates a compass.
+
+    To calibrate the compass, lie it flat and spin it horizontally. For best
```

Comment:
Minor grammatical question - should this be "lie it flat" or "lay it flat"? The latter sounds better to my ears, but I can't say I'm certain which is correct.